python/cudf/cudf/_lib/string_casting.pyx

```diff
@@ -509,7 +509,8 @@ def from_booleans(Column input_col):
 
 def int2timestamp(
         Column input_col,
-        format):
+        string format,
+        Column names):
     """
     Converting/Casting input date-time column to string
     column with specified format
```

Comment:
Add the newly added params to docstring here..

Reply:
Sounds good!!